File: test/unit/code_parser/whole_tests/views/createobject.rb
# Copyright (c) 2000 ActivePDF, Inc.
# ActivePDF DocConverter 2015
# Example generated 01/01/2000 

require 'win32ole'

# Instantiate Object
oDC = WIN32OLE.new("APDocConverter.Object")

# Release object: .NET = Yes, COM = Yes
oOne = WIN32OLE.new("APDocConverter.FromPDFOptions")
oOne.ToWordHeadersAndFootersMode = 0
oDC.SetFromPDFOptions(oOne)

# Release Object
oOne = ''

# Release object: .NET = No, COM = Yes
oTwo = WIN32OLE.new("APDocConverter.FromPDFOptions")
oTwo.ToWordHeadersAndFootersMode = 0
oDC.SetFromPDFOptions(oTwo)

# Release Object
oTwo = ''

# Release object: .NET = Yes, COM = Yes
oThree = WIN32OLE.new("APDocConverter.FromPDFOptions")
oThree.ToWordHeadersAndFootersMode = 0
oDC.SetFromPDFOptions(oThree)

# Release Object
oThree = ''

# Release Object
oDC = ''

# Process Complete
puts "Done!"

File: test/unit/code_parser/whole_tests/views/createobject.php
<?php
// Copyright (c) 2000 ActivePDF, Inc.
// ActivePDF DocConverter 2015
// Example generated 01/01/2000 
?>

<?php
// Instantiate Object
$oDC = new COM("APDocConverter.Object");

// Release object: .NET = Yes, COM = Yes
$oOne = new COM("APDocConverter.FromPDFOptions");
$oOne->ToWordHeadersAndFootersMode = 0;
$oDC->SetFromPDFOptions($oOne);

// Release Object
$oOne = null;

// Release object: .NET = No, COM = Yes
$oTwo = new COM("APDocConverter.FromPDFOptions");
$oTwo->ToWordHeadersAndFootersMode = 0;
$oDC->SetFromPDFOptions($oTwo);

// Release Object
$oTwo = null;

// Release object: .NET = Yes, COM = Yes
$oThree = new COM("APDocConverter.FromPDFOptions");
$oThree->ToWordHeadersAndFootersMode = 0;
$oDC->SetFromPDFOptions($oThree);

// Release Object
$oThree = null;

// Release Object
$oDC = null;

// Process Complete
echo "Done!";
?>